site/admin/theatres/index.php

```diff
@@ -0,0 +1,22 @@
+<html>
+<?php
+		include_once('init.php');
+		PrintHeader('St. Louis Theatre List', 'Theatres');
+	?>
+		<div class="TheatreList">
+			<?php
+				$TheatreList = TheatreListAlpha();
+				foreach ($TheatreList as $Row){
+					echo"<p class='Headline'>".$Row['TheatreName']."</p>";
```

Comment:
This is just a personal style preference, but I don't really see the need to make each line a separate echo statement. You could write this code a big cleaner by doing:
```
echo"
   <p class='Headline'>$Row[TheatreName]</p>
   <p>$Row[TheatreDescription]</p>
   ...
"
```